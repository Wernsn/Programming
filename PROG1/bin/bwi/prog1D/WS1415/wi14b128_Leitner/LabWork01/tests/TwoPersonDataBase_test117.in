1
Bugs
Bunny
20001301
20000001
19991232
20041131
20000431
20030230
20030229
19000229
19190303
3
0
