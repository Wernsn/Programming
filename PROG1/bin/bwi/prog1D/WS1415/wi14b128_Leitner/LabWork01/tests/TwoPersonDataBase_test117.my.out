Welcome to TwoPersonDataBase!

available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: enter firstname (1)enter lastname (1)please enter date of birth (1) (yyyymmdd)invalid month (13)
please enter date of birth (1) (yyyymmdd)invalid month (0)
please enter date of birth (1) (yyyymmdd)invalid day (32)
please enter date of birth (1) (yyyymmdd)invalid day (31)
please enter date of birth (1) (yyyymmdd)invalid day (31)
please enter date of birth (1) (yyyymmdd)invalid day (30)
please enter date of birth (1) (yyyymmdd)invalid day (29)
please enter date of birth (1) (yyyymmdd)invalid day (29)
please enter date of birth (1) (yyyymmdd)
available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: 
1: BUGS       BUNNY         Monday, 03/03/1919
available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: Thank you for using TwoPersonDataBase